go.mod

```diff
@@ -32,7 +32,7 @@ require (
 	github.com/armon/go-metrics v0.4.0
 	github.com/armon/go-radix v1.0.0
 	github.com/asaskevich/govalidator v0.0.0-20200907205600-7a23bdc65eef
-	github.com/aws/aws-sdk-go v1.44.95
+	github.com/aws/aws-sdk-go v1.44.126
```

Comment:
FWIW it looks like there is a slightly newer version now: https://github.com/aws/aws-sdk-go/releases/tag/v1.44.127

Reply:
Good catch, can get that in and retest before merging